NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
```

Comment:
Reply.

---
In reply to: [68509653](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68509653) [](ancestors = 68509653)